util/slack_wrapper.py

```diff
@@ -45,6 +45,13 @@ def set_purpose(self, channel, purpose, is_private=False):
         api_call = "groups.setPurpose" if is_private else "channels.setPurpose"
         return self.client.api_call(api_call, purpose=purpose, channel=channel)
 
+    def set_topic(self, channel, topic, is_private=False):
+        """
```

Comment:
One line doc string

Reply:
the other docstrings in the file are multiline, just fyi